src/components/strategies/common/BudgetInput.tsx

```diff
@@ -8,44 +8,62 @@ import { UseQueryResult } from '@tanstack/react-query';
 interface Props {
   id?: string;
   children?: ReactNode;
-  order: OrderCreate;
+  budgetValue: string;
+  budgetError?: string;
   token: Token;
-  query: UseQueryResult<string>;
+  query?: UseQueryResult<string>;
   onChange: (value: string) => void;
   disabled?: boolean;
   withoutWallet?: boolean;
   'data-testid'?: string;
+  title: string;
+  titleTooltip: string;
 }
 
 export const BudgetInput: FC<Props> = (props) => {
-  const { id, order, token, query, children, onChange } = props;
+  const {
+    id,
+    budgetValue,
+    budgetError,
+    token,
+    query,
+    children,
+    onChange,
+    title,
+    titleTooltip,
+  } = props;
   const inputId = useId();
-  const balance = query.data ?? '0';
+  const balance = query?.data ?? '0';
 
   return (
     <div className="flex flex-col gap-16">
+      <legend className="flex text-14 font-weight-500">
```

Comment:
legend must be used inside a <fieldset>
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/legend

Maybe use a label instead ?